Google.Api.Generator/Generation/SourceFileContext.cs

```diff
@@ -75,6 +75,13 @@ public IDisposable InNamespace(NamespaceDeclarationSyntax ns)
             return new Disposable(() => Namespace = restoreNamespace);
         }
 
+        public IDisposable InClass(ClassDeclarationSyntax cls)
```

Comment:
I'm having a hard time working out what this is trying to achieve. An XML comment would help. (Is it just a matter of saying "We're already in the given class, so we don't need to import it"?)